gradio/events.py

```diff
@@ -35,8 +39,7 @@ def change(
             status_tracker: StatusTracker to visualize function progress
             scroll_to_output: If True, will scroll to output component on completion
             show_progress: If True, will show progress animation while pending
-            _js: Optional frontend js method to run before running 'fn'. Input arguments for js method are values of input and outputs components, return should be a list of values for output component.
```

Comment:
Removing the underscore params from the docstring because our doc building code does not allow documenting them.

Reply:
I think we still need description of _js in the function for users who check the function to understand what it is clearly. Can we find a better solution? How does describing it just after docstring as a comment line?

Valid for postprocess and preprocess as well.

Reply:
Sounds good to me!